benchmark/benchmark/run_benchmark.py

```diff
@@ -32,6 +33,8 @@ def tool_map(tool):
         return prediction_request_sme
     elif tool in ["claude-prediction-offline", "claude-prediction-online"]:
         return prediction_request_claude
+    elif tool in ["prediction-request-rag"]:
```

Comment:
```suggestion
    elif tool == "prediction-request-rag":
```

However, this whole `tool_map` method could be a mapping instead. Then use `.get(name, None)` and if `None`, raise the Exception below.